src/popup/router/index.js

```diff
@@ -31,6 +31,7 @@ Vue.component('Loader', LoaderComponent);
 const router = new VueRouter({
   routes,
   mode: process.env.PLATFORM === 'web' ? 'history' : 'hash',
+  scrollBehavior: () => ({ x: 0, y: 0 }),
```

Comment:
this is fine, but let's also preserve position stored in history
```suggestion
  scrollBehavior: (to, from, savedPosition) => savedPosition || { x: 0, y: 0 },
```
https://router.vuejs.org/guide/advanced/scroll-behavior.html